src/java/org/apache/cassandra/db/guardrails/GuardrailsConfig.java

```diff
@@ -145,6 +145,8 @@
      */
     boolean getReadBeforeWriteListOperationsEnabled();
 
+    boolean getAllowFilteringEnabled();
```

Comment:
Missing javadoc (to be consistent w/rest of file, I think it's pretty self explanatory and redundant but "When In Rome"...)